vehicle_odom_topic: "/odom" 
vehicle_imu_topic: "/imu_raw"
vehicle_cmd_topic: "/vehicle_cmd" 
vehicle_state_topic: "/vehicle_state" 
target_speed: 5
goal_tolerance: 0.5
control_frequency: 20
vis_frequency: 0.5
path_vis_topic: "path_vis" 
speed_P: 0.4
speed_I: 0.1 
speed_D: 0 
frame_id: "gps" 

attack_start_index: 620
attack_end_index: 660
attack_mode: 1
recovery_start_index: 650
max_recovery_step: 100

save_state: true
debug: false